src/utils/postErrorToSlack.js

```diff
@@ -110,12 +108,14 @@ const postErrorToSlack = async (errorObject, reduxState) => {
   const timestamp = new Date().toISOString();
   const url = window.location.href;
   const experimentId = extractExperimentId(url);
+  const { networkResources: { environment } } = reduxState;
 
   const context = {
     user,
     timestamp,
     experimentId,
     url,
+    environment,
```

Comment:
instead of passing `environment` in here as an argument, why don't you move `const { networkResources: { environment } } = reduxState;` to `buildErrorMessage`?

Reply:
If the error message contains too much information, maybe we can remove the URL and User part of the message? We can still see those parts inside the preview.

Reply:
I prefer not to remove them from the message, because they are not clickable inside this text attachment. If we remove them, better to remove them from the text attachment. It is a recommendation, not compulsory change though.

Reply:
I understand your concern. I'll move the `== DETAILS ==` part below the `== ERROR ==` part so that we can see the error details at a glance without having to open the panel.